appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: '1.0.{build}'
+configuration:
+- Debug
+- Release
+platform: Any CPU
+environment:
+  # Don't report back to the mothership
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
+init:
+- ps: $Env:LABEL = "CI" + $Env:APPVEYOR_BUILD_NUMBER.PadLeft(5, "0")
```

Comment:
I think I'd prefer to just use the plain old version number.  When we append _CI_ it breaks the AssemblyInfo stamping.